packages/apps/storybook/.storybook/preview.js

```diff
@@ -37,7 +37,7 @@ export const parameters = {
   },
   authClientConfig: {
     clientId: process.env.STORYBOOK_AUTH_CLIENT_ID,
-    scope: ["imodels:read", "imodels:modify", "projects:read", "itwins:modify"].join(" "),
+    scope: ["imodels:read", "imodels:modify", "projects:read", "itwins:modify", "itwins:read"].join(" "),
```

Comment:
Unless you still call the project API somewhere, projects:read should not be required anymore.

Reply:
Done

Reply:
projects:read is still in there

Reply:
![image](https://user-images.githubusercontent.com/64415995/235910121-d786fc46-08f8-497a-9620-1bae7d15448b.png)

Maybe you are not on Removed projects:read commit?